src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -2,7 +2,13 @@
   "version": "1.0.0-preview3-*",
   "buildOptions": {
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../../src/dotnet/DotnetFiles.cs"
```

Comment:
When DotnetFiles tries to read a file that does not exist, it simply returns. It does not throw. But like I said, I think I have an alternative that does not require it.